frontend/src/metabase/parameters/components/ParameterSidebar/ParameterSidebar.tsx

```diff
@@ -152,16 +158,33 @@ export const ParameterSidebar = ({
       onRemove={handleRemove}
       data-testid="dashboard-parameter-sidebar"
     >
-      <SidebarHeader>
-        <Radio
-          value={tab}
-          options={tabs}
-          variant="underlined"
-          onChange={setTab}
-        />
-      </SidebarHeader>
-      <SidebarBody>
-        {tab === "settings" ? (
+      <Tabs
+        value={tab}
+        onTabChange={handleTabChange}
+        styles={{
+          tab: {
```

Comment:
Please don't override it like that, it won't work in v7. I'd suggest leaving it as it is or contacting the design team if we need to change the component

Reply:
<img width="889" alt="image" src="https://github.com/metabase/metabase/assets/125459446/db4835ae-f51a-4ca4-9a9b-cc8ff22b5bf2">

as I understood from https://mantine.dev/guides/6x-to-7x/#styles-prop only nesting of selectors doesn't work, so I presume mantine will not expand `&` with a className in `styles` prop, but it will still work.

I can try to rework it with class if you're sure that `styles` will not work in v7, but we already have several places using styles anyway https://github.com/search?q=repo%3Ametabase%2Fmetabase+styles%3D%7B&type=code

Reply:
Are we sure that's not going to work?
https://mantine.dev/guides/6x-to-7x/#styles-prop
Says that `nested selectors` won't work, but I think what they mean is something like 
<img width="237" alt="image" src="https://github.com/metabase/metabase/assets/1914270/e6726b1c-fb64-482d-abb8-42428475475c">
They say that "Regular selectors are still supported":
<img width="511" alt="image" src="https://github.com/metabase/metabase/assets/1914270/81a2e209-c6a1-48fb-8739-f5ca592b4998">

Reply:
okay, waiting for design feedback https://metaboat.slack.com/archives/C02H619CJ8K/p1712668558824799